src/search-schema/data/definitions/enums/organ_types.yaml

```diff
@@ -28,8 +32,10 @@ HT:
   iri: http://purl.obolibrary.org/obo/UBERON_0000948

```

Comment:
Per Mariah's comment in the Slack thread, I suggest using the UBERON entries for [right uterine tube](http://purl.obolibrary.org/obo/UBERON_0001302) and [left uterine tube](http://purl.obolibrary.org/obo/UBERON_0001303) for the left and right fallopian tubes.

Reply:
@pdblood @computationdoc similar inquiry as before. What is the proper/correct entry to be used?

Reply:
Updated as per @bhonick and @MariahKenney suggestion.